src/validations/routine.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import Joi from 'joi';
+
+import { CustomError } from 'src/types/custom-error';
+
+export const routineSchema = Joi.object({
+  id: Joi.number().integer(),
```

Comment:
I think that the ID is not required to be sent when you create the routine. WDYT?